Documentation/Storage-Configuration/NFS/nfs-security.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Security
+---
+
+## User ID management
+
+!!! attention
+    This feature is experimental and will not support upgrades to future versions.
```

Comment:
Do we know of something that would make upgrades in the future difficult? If it might work, perhaps we soften the wording.
```suggestion
    This feature is experimental and may not support upgrades to future versions.
```

Reply:
Softening the wording seems good. I think it's likely we will be able to upgrade just fine, but there could be some breaking API changes if we discover something critical.